kiosk/src/Kiosk.css

```diff
@@ -1,98 +1,235 @@
+    width: 100vw;
+    height: 100vh;
 }
 
 h1 {
     font-weight: 400;
-    font-size: 54px;
-    line-height: 54px;
+    font-size: 3.375em;
+    line-height: 3.375rem;
     text-align: center;
 }
 
 h2 {
     font-weight: 400;
-    font-size: 44px;
-    line-height: 44px;
+    font-size: 2.75em;
+    line-height: 2.75rem;
     text-align: center;
-    color: white;
 }
 
-.mainMenuHighScoreHeader {
+.sim-embed {
+    width: 100%;
+    height: 100%;
+    top:0;
+    left: -2vw;
+    position: absolute;
+}
+
+.footer {
+    position: fixed;
+    /* These numbers are relative to the size of the footer.png logo */
+    height: 5vh;
+    width: 20vw;
+    left: 1.5%;
+    bottom: 0;
+    margin-bottom: 2vh;
+    background: url("footer.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+}
+
+/* -------- MAIN MENU SYTLING -------- */
+/* ----------------------------------- */
+
+.mainMenu {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+.mainMenuHeader {
     font-weight: 400;
-    font-size: 40px;
-    line-height: 40px;
-    text-align: center;
+    font-size: 2.5rem;
     color: white;
 }
 
-.highScoreSmallDiv {
+.mainMenuTopBar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    margin-left: 28.5vw;
+}
+
+.kioskButton {
+    width: 20em;
+    border-color: lightblue;
+    background-color: transparent;
+    font-family: inherit;
+    padding: 2em 0.5em;
+    color:white;
+    border-radius: 0.75em;
+    border-width: 0.2em;
+    border-style: solid;
+}
+
+/* -------- ADDING GAME PAGE SYTLING -------- */
+/* ----------------------------------- */
+
+.addInstructions h2 {
+    color: lightblue;
+    font-size: 2em;
+    /* margin-left: -7em; */
+}
+
+.addInstructions li {
+    padding: 1em 0;
+    margin-left: 1.5rem;
+    font-size: large;
+}
+
+.addInstructions {
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+}
+
+.addGame h1 {
+    font-size: 2.5em;
     font-weight: 400;
-    font-size: 32px;
-    line-height: 32px;
-    text-align: center;
-    color: #8AFFF8;
+    margin: 1em 0;
 }
 
-.footer {
+.addGame {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100vw;
+    height: 100vh;
+}
+
+.innerQRCodeContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.addGameContent {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    margin-top: 5vw;
+}
+
+
+
+/* -------- QR PAGES SYTLING -------- */
+/* ----------------------------------- */
+
+.scanQrPage {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 100vw;
+    max-height: 100vh;
+}
+
+.scanQrPage h2 {
+    margin-bottom: 0;
+}
+
+.scanQrButton {
+    width: 20em;
+    border: none;
+    background-color: rgba(36, 122, 160, 0.81);
```

Comment:
instead of rgba, specify the color as a hex string and use the opacity property. same below